index.html

```diff
@@ -0,0 +1,126 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/bf723a76a7.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="Css/style.css">
+    <title>Document</title>
+</head>
+<body>
+    <header>
+        <nav class="mobile">
+            <a href="#">
+            <div class="hamburger">
+                <span class="bars"></span>
+                <span class="bars"></span>
+                <span class="bars"></span>
+            </div>
+            </a>
+            <ul class="disappear" id="navList">
+                <li><a href="index.html">Home</a></li>
+                <li><a href="about.html">About Us</a> </li>
+                <li><a href="#">Programs</a></li>
+                <li><a href="#">Join</a> </li>
+                <li><a href="#">Sponsors</a> </li>
+                <li><a href="#">members</a> </li>
+            </ul>
+        </nav>
+        <nav class="Desktop">
+            <ul class="social-media">
+                <li><i class="fa-brands fa-facebook-f"></i></li>
+                <li><i class="fa-brands fa-twitter"></i></li>
+                <li>English</li>
+                <li>My page</li>
+                <li>logout</li>
+            </ul>
+            <div class="full-nav">
+                <h3>GSN</h3>
+                <ul class="navigation">
+                    <li><a href="index.html">Home</a></li>
+                    <li><a href="about.html">About Us</a> </li>
+                    <li>Programs</li>
+                    <li>Join</li>
+                    <li>Sponsors</li>
+                    <li class="red-member">members</li>
+                </ul>
+            </div>
+        </nav>
+    </header>
+    <section id="About">
+        <h1>Global Students Network</h1>
+        <p>A joyful celebration believing in the value of openness and sharing, creating a positive change with students from all
+        over the world</p>
+        <h2 id="hot-date">2022.12.20(tue) - (fri)</h2>
+        <h3 id="event">@student exchange</h3>
+    </section>
+    <section id="programs">
```

Comment:
- [ ] As previously requested kindly ensure this section has only five programs as shown In the design given, kindly remove the extra projects and also make sure the design matches the given design